src/main/java/RebornStorage/tiles/CraftingNode.java

```diff
@@ -0,0 +1,170 @@
+package RebornStorage.tiles;
+
+import RebornStorage.init.ModBlocks;
+import com.raoulvdberge.refinedstorage.api.autocrafting.ICraftingPattern;
+import com.raoulvdberge.refinedstorage.api.autocrafting.ICraftingPatternContainer;
+import com.raoulvdberge.refinedstorage.api.autocrafting.ICraftingPatternProvider;
+import com.raoulvdberge.refinedstorage.api.network.INetwork;
+import com.raoulvdberge.refinedstorage.api.network.node.INetworkNode;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import net.minecraftforge.items.IItemHandler;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class CraftingNode implements INetworkNode, ICraftingPatternContainer {
+
+	TileMultiCrafter multiCrafter;
```

Comment:
Not allowed! Pass a world and a blockpos instead of the tile.

Do not access the tile in here. The inventory **has** to be saved in the CraftingNode.